core/interfaces.go

```diff
@@ -65,7 +65,7 @@ type RegistrationAuthority interface {
 	NewCertificate(ctx context.Context, csr CertificateRequest, regID int64, issuerNameID int64) (Certificate, error)
 
 	// [WebFrontEnd]
-	UpdateRegistration(ctx context.Context, base, updates Registration) (Registration, error)
+	UpdateRegistration(ctx context.Context, base, updates *corepb.Registration) (*corepb.Registration, error)
```

Comment:
It doesn't need to happen in this PR necessarily, but this interface is going to need to convert to
```
UpdateRegistration(ctx context.Context, req *rapb.UpdateRegistrationRequest) (*corepb.Registration, error)
```
before the unwrapping is 100% complete.

Reply:
Sounds good. I think I will add that as a follow up change.